.changeset/hot-ghosts-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/dev-utils': patch
```

Comment:
Nit: Should this be a minor? Seems like a new feature, rather than a bug fix.

> MINOR version when you add functionality in a backward compatible manner
> PATCH version when you make backward compatible bug fixes

Source: https://semver.org/